animations/concepts.md

```diff
@@ -0,0 +1,400 @@
+<img src="/animations/images/easing.png" name="Easing" height="100%" width="100%">
+</a>
+
+
+This graph shows the easing of an element that remains visible throughout a transition. The y-axis shows the position and the x-axis shows the time.
+
+
+<!-- <div>
+<table class="table" width="100%">
+  <col width="55%">
+  <col width="45%">
+	<tbody>
+    <tr>
+    <td>
+    <a href="" onMouseOver="document.Easing.src='/animations/images/easing.gif';" onMouseOut="document.Easing.src='/animations/images/easing.png';">
+    <img src="/animations/images/easing.png" name="Easing">
+    </a></td>
+      <td>
+      <p>Easing is a way to adjust an animation’s rate of change—easing allows transitioning elements to speed up and slow down, rather than moving at a constant rate. In the real world, things don’t start or stop moving instantaneously. They take time to speed up and slow down. Easing makes elements move as though influenced by natural forces like friction.</p>
+      <p>This graph shows the easing of an element that remains visible throughout a transition.</p>
+      <p>The y-axis shows the position and the x-axis shows the time.</p>
+      </td>
+
+    </tr>
+   </tbody>
+  </table>
+</div> -->
+
+You can also find helpful information about using  [Material](/widgets/material/) components on the Flutter website.  
+
+### Dart language basics
+Although you don't need to be a Dart programmer to create a Flutter mobile app, you can find helpful information on the [Dart programming language](https://www.dartlang.org/) website and in the [Dart Language Tour](https://www.dartlang.org/guides/language/language-tour).  
+
+For example, Dart uses cascade (`..`) notation. Method cascading is a syntax that calls multiple methods on the same object. The example below shows `..addListener()` which means that the `addListener()`
+method is called with the return value from `animate()`.  
+
+<!-- skip -->
+{% prettify dart %}
+```Dart
+[[highlight]]animation = tween.animate(controller)[[/highlight]]
+          [[highlight]]..addListener(()[[/highlight]] {
+            setState(() {
+              // the animation object’s value is the changed state
+            });
+          });
+```
+{% endprettify %}  
+
+This code is equivalent to:  
+
+<!-- skip -->
+{% prettify dart %}
+```Dart
+[[highlight]]animation = tween.animate(controller);[[/highlight]]
+[[highlight]]animation.addListener(()[[/highlight]] {
+            setState(() {
+              // the animation object’s value is the changed state
+            });
+          });
+```
+{% endprettify %}
+
+In addition to function calls, cascade notation allows you to access fields on that same object. This often saves you the step of creating a temporary variable and allows you to write more fluid code.
+
+You can learn more about cascade notation in the
+[Dart Language Tour.](https://www.dartlang.org/guides/language/language-tour)
+
+
+## Animation<T\> class
```

Comment:
I think you need to escape the < and > here, because it's getting treated like an HTML tag.